README.md

```diff
@@ -106,7 +106,9 @@ All tests are found under the tests directory in the cit-api and should be added
 
 ## Deployment
 
-#todo when worked out.
+Create a Pull request on [https://github.com/bcgov/CIT](https://github.com/bcgov/CIT), once its approved and merge by a repo admin, it will automatically be deployed to test.
```

Comment:
```suggestion
Create a Pull request on [https://github.com/bcgov/CIT](https://github.com/bcgov/CIT), once its approved and merged by a repo admin, it will automatically be deployed to test.
```